src/Session.hs

```diff
@@ -81,11 +81,15 @@ kill ghci = ignored $ do
     -- See: https://github.com/ndmitchell/ghcid/issues/254
     showCursor
 
-loadedModules :: [Load] -> [FilePath]
-loadedModules = nubOrd . map loadFile . filter (not . isLoadConfig)
+loadedModules :: FilePath -> [Load] -> [FilePath]
+loadedModules dir = nubOrd . map (loadFile . qualify dir) . filter predicate
+    where
+      predicate Message{loadFile = loadFile} = loadFile /= "<unknown>"
+      predicate Loading{loadFile = loadFile} = loadFile /= "<unknown>"
```

Comment:
Is there a nice way to combine these cases?

Reply:
I don't think so.

Reply:
Or patterns will help here